Microsoft/pyproject.toml

```diff
@@ -0,0 +1,90 @@
+[tool.poetry]
+name = "Sekoia.io Automation Module to work with Microsoft Server Actions events"
+version = "0.0"
+description = ""
+authors = []
+
+[tool.poetry.dependencies]
+python = ">=3.10,<3.12"
+orjson = "^3.7.7"
+python-dateutil = "^2.8.2"
+sekoia-automation-sdk = { version = "^1.8.1", extras = ["all"] }
+async-lru = "^2.0.2"
+aiohttp = "^3.8.5"
+aiolimiter = "^1.1.0"
+pywinrm = {extras = ["credssp"], version = "^0.4.3"}
+
+[tool.poetry.dev-dependencies]
+pytest = "*"
+pytest-cov = "*"
+pytest-asyncio = "*"
```

Comment:
Same

Reply:
By default I use `@pytest.mark.asyncio` in tests to speed them up a bit